### PR TITLE
Create mobile menu functionality

### PR DESCRIPTION
**Pull request by aceDavon**
Add javascript event listener for click to display the mobile menu and
hide on click toggle

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,32 +14,60 @@
...
         <li class="top-menu-item motion">
           <a href="#" class="links"
             ><img src="./img/icons/email.png" alt="mail-icon"
           /></a>
         </li>
       </ul>
+      <div class="menu-container" id="topMenu">
+        <div class="close" id="close">
+          <img src="./img/icons/close_white_48dp.svg" alt="close-icon" />
+        </div>
```
Comment (youmari) [inline index.html:43]: - [ ] please make sure that the close button has the same size as the one in the Figma design and also try to align it in the right place as in the Figma design 
image

File: index.html [hunk 2]
```diff
@@ -14,32 +14,60 @@
...
+              alt="pointer2"
+            />
+          </li>
+          <li>
+            <a href="#contact">Contact</a>
+            <img
+              src="./img/icons/keyboard_arrow_right_white_48dp.svg"
+              alt="pointer3"
+            />
+          </li>
```
Comment (youmari) [inline index.html:65]: - [ ] please try to use the same font-size, font family, space between the links in order to match the Figma design 
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @aceDavon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great title and short summary 
✔️ No linter error
✔️ Github flow 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
Hello @aceDavon and @SurafelGetachewT  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Made changes according to previous reviewer 👍 
## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**